lib/pbench/agent/__init__.py

```diff
@@ -21,24 +21,24 @@ def __init__(self, cfg_name):
 
         try:
             # Provide a few convenience attributes.
-            self.agent = self.conf["pbench-agent"]
-            self.results = self.conf["results"]
+            self.agent = self._conf["pbench-agent"]
+            self.results = self._conf["results"]
             # Now fetch some default common pbench settings that are required.
             self.pbench_run = Path(
-                self.conf.get(
+                self.get(
                     "pbench-agent", "pbench_run", fallback=DEFAULT_PBENCH_AGENT_RUN_DIR
                 )
             )
```

Comment:
Given that we're using `self.get()` at line 28 (et al.), shouldn't we be using it at lines 24 and 25, as well?

Reply:
Those are not the same as a `.get()`, see https://docs.python.org/3.9/library/configparser.html#mapping-protocol-access.

Reply:
Ah, right!  Thank you.

Can you tell whether these two attributes are actually referenced?  (I have the impression that they are not...in which case perhaps we could remove them?)

Reply:
I did not bother to look.  Further changes like that are beyond the scope of this PR.

Reply:
FWIW - When I made that change in another branch off of `main`, various agent python unit tests failed, and the agent legacy tests failed in various ways which did not immediately look related to the change.  I am running Fedora 37 locally, which might be the cause of some or all of the legacy tests failures.

Reply:
See PR #3226 for further work.